chirp/drivers/ga510.py

```diff
@@ -667,7 +858,7 @@ def apply_sk_listvalue(setting, obj):
                                                     SK_CHOICES[idx]))
             rs.set_apply_callback(apply_sk_listvalue, _skey.skey2lp)
             adv.append(rs)
-
+            
```

Comment:
Please don't introduce random whitespace damage, and especially don't add trailing whitespace. A good portion of the errors in the style check are because you're adding whitespace. I won't mark all the rest of them here, because the style check is doing it for me.